packages/core/admin/admin/src/pages/ProfilePage/index.js

```diff
@@ -73,6 +75,7 @@ const ProfilePage = () => {
   const { lockApp, unlockApp } = useOverlayBlocker();
   const { notifyStatus } = useNotifyAT();
   const { currentTheme, themes: allApplicationThemes, onChangeTheme } = useThemeToggle();
+  const { get } = useFetchClient();
   useFocusWhenNavigate();
 
   const { status, data } = useQuery('user', () => fetchUser(), {
```

Comment:
```suggestion
  const { status, data } = useQuery('user', fetchUser, {
```

This should be ample no?

Reply:
done